debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/RecordsSnapshotProducerIT.java

```diff
@@ -434,7 +435,7 @@ public void shouldGenerateSnapshotsForPartitionedTables() throws Exception {
 
         // verify each topic contains exactly the number of input records
         assertEquals(1, topicCounts.get("test_server.public.first_table").intValue());
-        assertEquals(30, topicCounts.get("test_server.public.partitioned").intValue());
```

Comment:
@jpechane, WDYT about this one? I think it's ok to include this change in a minor; I'd argue it makes more sense that way actually. But it also settles that we shouldn't add this in 1.1.x IMO.

Reply:
@gunnarmorling That's definiteyl breaking compatibility change so no way adding it to 1.1.x. IIUC this now aligns Postres with MySQL and we must warn the user to use `ByLogicalTableRouter` to get the functionality back.

Reply:
Hum, hum. So is MySQL indeed creating one topic per partition? Wondering whether it'd not make more sense to write all partitions to a single topic actually; is the information on partitioning relevant for downstream consumers?

Reply:
@gunnarmorling I stand corrected. MySQL partitioning is about spreading the table into different physical files. POstgresSQL partitioning is about spreading to different tables. MySQL equivalent is sharding.

Reply:
Ok, so what number of topics do we get for a partitioned table in MySQL? Is it the same as for Postgres with the driver update?

Reply:
@gunnarmorling If I understand this change then for n paritions and one aggregate table we get
1) n tables for PostgreSQL partitioned table
2) 1 table for MySQL partitioned table
3) n table for MySQL sharded tables

As PostgreSQL partitioning seems to be equivalent to MySQL sharding